packages/terra-form/src/Select.scss

```diff
@@ -0,0 +1,11 @@
+@import '~terra-mixins';
+
+.terra-Form-select {
+  background-color: #fff;
+  border: #dedfe0 solid 1px;
+  border-radius: 0.25em;
+  font-size: 1em;
+  line-height: 1.428em;
```

Comment:
We want to use unitless line-height values. 

https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0